articles/integrations/aws/index.md

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
Good idea adding an index! Can you convert this to the `index.html` format though? It looks a bit better that way :)